app/controllers/podcast_appearances_controller.rb

```diff
@@ -0,0 +1,123 @@
+class PodcastAppearancesController < ApplicationController
```

Comment:
Hi @NenadPantelic, thanks for the changes. I recommend, see also the title of this PR, to separate the the table code from the controller code (policy included) and submit the latter separately once the table is merged. Also because this controller doesn't have neither tests nor templates.